lib/ncr_dispatcher.rb

```diff
@@ -0,0 +1,6 @@
+class NcrDispatcher < LinearDispatcher
```

Comment:
I'm happy leaving this ncr-specific for now, although user-specific is the more realistic use case if/when we start managing user preferences. The main problem that this would need to solve for now is that Navigator folks are happy with sending an email to requesters every time an approval happens, while the NCR folks are not.